src/drivers/gps/params.c

```diff
@@ -122,12 +122,30 @@ PARAM_DEFINE_INT32(GPS_UBX_MODE, 0);
  * @max 360
  * @unit deg
  * @reboot_required true
- * @decimal 0
+ * @decimal 3
  *
  * @group GPS
  */
 PARAM_DEFINE_FLOAT(GPS_YAW_OFFSET, 0.f);
 
+/**
+ * Pitch offset for dual antenna GPS
+ *
+ * Vertical offsets can be compensated for by adjusting the Pitch offset (Septentrio).
+ *
+ * Note that this can be interpreted as the "roll" angle in case the antennas are aligned along the perpendicular axis. This occurs in situations where the two antenna ARPs may not be exactly at the same height in the vehicle reference frame. Since pitch is defined as the right-handed rotation about the vehicle Y axis, a situation where the main antenna is mounted lower than the aux antenna (assuming the default antenna setup) will result in a positive pitch.
+ *
+ *
+ * @min -90
+ * @max 90
+ * @unit deg
+ * @reboot_required true
+ * @decimal 3
+ *
+ * @group GPS
+ */
+PARAM_DEFINE_FLOAT(GPS_PITCH_OFFSET, 0.f);
```

Comment:
Do you also plan to update the EKF to use that parameter in this PR?

Reply:
That is the plan, but for now I will leave it like this.